readme.md

```diff
@@ -18,13 +18,13 @@ $ npm install --save-dev gulp-babel babel-preset-es2015
 const gulp = require('gulp');
 const babel = require('gulp-babel');
 
-gulp.task('default', () => {
-	return gulp.src('src/app.js')
```

Comment:
Why have the return statements been removed? I believe the stream must be returned for the task to be aware of when it ends. If the stream is not returned, then the task will mark itself as done without waiting for operations to finish.